apps/web/src/components/NftSelector/NftSelectorFilter/NftSelectorViewSelector.tsx

```diff
@@ -1,9 +1,10 @@
-import { useCallback, useState } from 'react';
+import { useCallback, useMemo, useState } from 'react';
```

Comment:
ensures the `Created` tab is disabled for non-mainnet options

<img width="430" alt="Screenshot 2023-09-21 at 3 23 58 PM" src="https://github.com/gallery-so/gallery/assets/12162433/c4853787-986b-4ccf-99e2-28ce1a1cfb2d">